### ISSUE TITLE
Some class not working in hierapolis

### ISSUE THREAD

**Issue by adamdilek** [yakut-project/yakut#25]
![screen shot 2013-09-10 at 2 52 28 pm](https://f.cloud.github.com/assets/989759/1114080/9793bab6-1a0f-11e3-8a5a-487a6f71acd3.png)

**Comment by onurozgurozkan** [comment]
@muhammetdilek  Is it finished?

**Comment by adamdilek** [comment]
Yes @onurozgurozkan. Completed in https://github.com/yakut-project/yakut/issues/29
